packages/framework-integration-tests/src/entities/car-model.ts

```diff
@@ -0,0 +1,27 @@
+import { Entity, Reduces } from '@boostercloud/framework-core'
+import { UUID } from '@boostercloud/framework-types'
+import { CarModelAdded } from '../events/car-model-added'
+import { CarModelNameUpdated } from '../events/car-model-name-updated'
+
+@Entity({
+  authorizeReadEvents: 'all',
+})
+export class CarModel {
+  public constructor(readonly id: UUID, public name: string, readonly brand: string) {}
+  public getId() {
+    return this.id
+  }
+  @Reduces(CarModelAdded)
+  public static addCarModel(event: CarModelAdded, currentCarModel: CarModel): CarModel {
+    return new CarModel(event.id, event.name, event.brand)
+  }
+
+  @Reduces(CarModelNameUpdated)
+  public static updateName(event: CarModelNameUpdated, currentCarModel: CarModel): CarModel {
+    if (!currentCarModel) {
```

Comment:
can we rename it as `reduceCarModelNameUpdated`? . If you think it would be too long, I think `nameUpdated` suits well too.